### PR TITLE
mtd_spi_nor: erase timings in struct

### PR DESCRIPTION
**Pull request by bergzand**
### Contribution description

This PR moves the defines from RIOT-OS#9349 to the mtd_spi_nor_t struct to be able to:

- configure the timings per NOR flash chip.
- Make it a bit more clear to the developer that these should be configured per board/chip.

### Testing procedure

Run your favorite MTD SPI NOR test. Please make sure to use a board that has an MTD SPI NOR device (Mulle, PineTime (with #13161) etc…)

### Issues/PRs references

Follow-up to #9349

### CODE REVIEW

File: boards/mulle/board.c [hunk 1]
```diff
@@ -61,6 +61,10 @@ static mtd_spi_nor_t mulle_nor_dev = {
     .addr_width = 3,
     .mode = SPI_MODE_3,
     .clk = SPI_CLK_10MHZ,
+    .wait_chip_erase = 16 * US_PER_SEC,
```
Comment (kaspar030) [inline boards/mulle/board.c]: Isn't this defaulting to `int` as result for the calculation? (change to `16LU * ...`)?

Comment (bergzand) [reply boards/mulle/board.c]: Changed all values in the struct for consistency

File: boards/mulle/board.c [hunk 2]
```diff
@@ -61,6 +61,10 @@ static mtd_spi_nor_t mulle_nor_dev = {
     .addr_width = 3,
     .mode = SPI_MODE_3,
     .clk = SPI_CLK_10MHZ,
+    .wait_chip_erase = 16LU * US_PER_SEC,
+    .wait_sector_erase = 40LU * US_PER_MS,
+    .wait_32k_erase = 20LU *US_PER_MS,
```
Comment (kaspar030) [inline boards/mulle/board.c]:
```suggestion
    .wait_32k_erase = 20LU * US_PER_MS,
```

**Comment by kaspar030** [comment]
please rebase and squash (and mayne commit the space fix)

**Comment by bergzand** [comment]
With both #13161 and #13332 merged, I've rebased and refactored this PR a bit to store the erase timings in the params struct.

File: boards/mulle/board.c (hunk 2 above)
Comment (bergzand) [reply boards/mulle/board.c]: Thanks, added

**Comment by kaspar030** [comment]
pls squash

**Comment by bergzand** [comment]
Squashed!

**Comment by kaspar030** [comment]
re-ACK.

**Pull request merged**